tests/lib/tools/query-mondodb

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+import sys
+
+from pymongo import MongoClient
+
+
+def _make_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--host', help="Hostname or ip for the mongo database",
+                        **environ_or_required('MONGO_HOST'))
+    parser.add_argument('--port', help="Port for the mongo database",
+                        **environ_or_required('MONGO_PORT'))
+    
+    parser.add_argument('--db-name', help="Name for the database used to insert", default="snapd")
+    parser.add_argument('--db-collection', help="Name for the collection used to insert", default="spread_errors")
+    parser.add_argument('--after-date', help="Find results after a date (yyyy-mm-dd)", default="2000-01-01")
+    parser.add_argument('--after-time', help="Find results after a time (hh:mm:ss)", default="00:00:00")
+    parser.add_argument('--before-date', help="Find results before a date (yyyy-mm-dd)", default="2999-12-12")
+    parser.add_argument('--before-time', help="Find results before a time (hh:mm:ss)", default="24:00:00")
```

Comment:
Usability question: would it be more practical to use current date/time as the default "before", and then 1 month back from current as the default value of "after"? So running it without after/before args would by default show data for last month?

Reply:
IMHO, this depends on how much data we expect, we are running a few thousand tests every day, so a full months worth of all tests may be way too much data by default? Maybe the default should just be the last 24 hours or even just the last 6 hours?

Reply:
Yes, fair point, my main concern is that 2000-01-01 is too much ;)

Reply:
Ah good point yes since 2000 will definitely be too much :-)

Reply:
Thanks for taking a look to this, by default it just displays 1000 values, it is the default for --limit, So if you make a search will get the last 1000 elements saved

Reply:
Ah sorry I missed the --limit option, could you document whether the default behavior for --limit is the most recent (newest) 1000 items or the least recent (oldest) 1000 items?